galley/pkg/crd/validation/webhook.go

```diff
@@ -103,6 +105,27 @@ type WebhookParameters struct {
 	EnableValidation bool
 }
 
+type createInformerWebhookSource func(cl clientset.Interface, name string) cache.ListerWatcher
+type createInformerEndpointSource func(cl clientset.Interface, namespace, name string) cache.ListerWatcher
+
+var (
+	defaultCreateInformerWebhookSource = func(cl clientset.Interface, name string) cache.ListerWatcher {
+		return cache.NewListWatchFromClient(
+			cl.AdmissionregistrationV1beta1().RESTClient(),
+			"validatingwebhookconfigurations",
+			"",
+			fields.ParseSelectorOrDie(fmt.Sprintf("metadata.name=%s", name)))
+	}
+
+	defaultCreateInformerEndpointSource = func(cl clientset.Interface, namespace, name string) cache.ListerWatcher {
+		return cache.NewListWatchFromClient(
+			cl.CoreV1().RESTClient(),
+			"endpoints",
+			namespace,
+			fields.ParseSelectorOrDie(fmt.Sprintf("metadata.name=%s", name)))
+	}
```

Comment:
We can eliminate the above two `ListerWatcher` constructor by `NewSharedInformerFactory`

Reply:
Oh, I missed ParseSelector. Ingor